static/src/javascripts/projects/common/modules/analytics/analytics.js

```diff
@@ -1,254 +0,0 @@
-        }
-
-        // not all pages have a production office
-        if (config.page.productionOffice) {
-            s.prop64 = config.page.productionOffice;
-        }
-
-        s.prop65    = config.page.headline || '';
-        s.eVar70    = config.page.headline || '';
-
-        if (s.getParamValue('INTCMP') !== '') {
-            s.eVar50 = s.getParamValue('INTCMP');
-        }
-        s.eVar50 = s.getValOnce(s.eVar50, 's_intcampaign', 0);
-
-        // the operating system
-        s.eVar58 = navigator.platform || 'unknown';
-
-        // the number of Guardian links inside the body
-        if (config.page.inBodyInternalLinkCount) {
-            s.prop58 = config.page.inBodyInternalLinkCount;
-        }
-
-        // the number of External links inside the body
-        if (config.page.inBodyExternalLinkCount) {
-            s.prop69 = config.page.inBodyExternalLinkCount;
-        }
-
-        s.prop75 = config.page.wordCount || 0;
-        s.eVar75 = config.page.wordCount || 0;
-
-        s.prop19 = 'frontend';
-        s.prop67    = 'nextgen-served';
-
-        // Set Page View Event
-        s.events    = s.apl(s.events, 'event4', ',', 2);
-
-        s.prop56    = guardian.isEnhanced ? 'Javascript' : 'Partial Javascript';
-
-        /* Set Time Parting Day and Hour Combination - 0 = GMT */
-        s.prop20    = tpA[2] + ':' + tpA[1];
-        s.eVar20    = 'D=c20';
-
-        /*
-         eVar1 contains today's date
-         in the Omniture backend it only ever holds the first
-         value a user gets, so in effect it is the first time
-         we saw this user
-         */
-        s.eVar1 = now.getFullYear() + '/' + pad(now.getMonth() + 1, 2) + '/' + pad(now.getDate(), 2);
-
-        s.prop7     = webPublicationDate ? new Date(webPublicationDate).toISOString().substr(0, 10).replace(/-/g, '/') : '';
-
-        if (webPublicationDate) {
-            s.prop30 = 'content';
-        } else {
-            s.prop30 = 'non-content';
-        }
-
-        s.prop47    = config.page.edition || '';
-
-        var userFromCookie = window.guardian.config.user;
-
-        if (userFromCookie) {
-            s.prop2 = 'GUID:' + userFromCookie.id;
-            s.eVar2 = 'GUID:' + userFromCookie.id;
-        }
-
-        s.prop31    = userFromCookie ? 'registered user' : 'guest user';
-        s.eVar31    = userFromCookie ? 'registered user' : 'guest user';
-
-        try {
-            var navInteractionData = window.sessionStorage.getItem(NG_STORAGE_KEY);
-
-            var navInteraction = JSON.parse(navInteractionData);
-
-            if (navInteraction) {
-                trackNavigationInteraction(navInteraction.value);
-                window.sessionStorage.removeItem(NG_STORAGE_KEY);
-            }
-
-        } catch (e) { }
-
-        // Sponsored content
-        if(window.guardian.isModernBrowser) {
-            var contentNodes = document.querySelectorAll('[data-sponsorship]');
```

Comment:
@mkopka Do you know if we need to recreate this for GA? Tracking sponsored content?